src/Spd.Presentation.Dynamics/Controllers/PrintingController.cs

```diff
@@ -43,19 +43,16 @@ public class PrintingController(IMediator mediator, IMapper mapper) : SpdControl
     }
 
     /// <summary>
-    /// input jobId, this endpoint will return the job status.
+    /// input eventId, this endpoint will get jobId from event queue and then query the job status to bc mail plus. After that, it will
+    /// update the event queue..
     /// </summary>
-    /// <param name="jobId"></param>
+    /// <param name="eventId"></param>
     /// <param name="ct"></param>
     /// <returns></returns>
-    [HttpGet("api/printjobs/{jobId}/status")]
-    public async Task<PrintJobStatusResponse> GetPrintJobStatus(string jobId, CancellationToken ct)
+    [HttpGet("api/printjobs/{eventId}/status")]
+    public async Task<Results<Ok<string>, BadRequest>> GetPrintJobStatus(Guid eventId, CancellationToken ct)

```

Comment:
How do we know the status if we only get the id?

Reply:
that's a good question. Actually, we update the status in the spd_eventqueue table.
But I think I should change here is return the status.
Let me fix that. thx.